model/model_test.go

```diff
@@ -1,9 +1,102 @@
+// Copyright 2019 The casbin Authors. All Rights Reserved.
```

Comment:
Put them into a new `config_test.go` file.

Reply:
I think you're referring to the `MockConfig` I've defined here.  I was using that to isolate the tests without using a complete `config.ConfigInterface` implementation.  If that gets moved to `config_test.go` it no longer becomes usable in this test.  I haven't found a consistent pattern for where people place their mocks in Go.  Would moving the Mock definitions to the bottom of the file be better?